ingestion_server/ingestion_server/indexer.py

```diff
@@ -522,6 +522,9 @@ def create_and_populate_filtered_index(
             },
             slices="auto",
             wait_for_completion=True,
+            # 10k derived from in-production testing
+            # See https://github.com/WordPress/openverse/issues/2963
+            requests_per_second=10_000,
```

Comment:
Actually, I'd say we should try 12k or 15k to speed up the process and prevent the timeout. It was under 50% of CPU utilization with 10k requests.
```suggestion
            requests_per_second=15_000,
```